ansible/config_sonic_basedon_testbed.yml

```diff
@@ -35,6 +35,81 @@
   tasks:
 
   - block:
+    - name: Creates telemetry directory
+      file:
+          path: /etc/sonic/telemetry
+          state: directory
+      become: true
+
+    - name: Init telemetry keys
+      set_fact:
+          server_key: ""
+          server_csr: ""
+          dsmsroot_key: ""
+          dsmsroot_csr: ""
+
+    - name: read server key
+      set_fact:
+          server_key: "{{ telemetry_certs['server_key'] }}"
+      when: telemetry_certs['server_key'] is defined
+
+    - name: read server csr
+      set_fact:
+          server_csr: "{{ telemetry_certs['server_csr'] }}"
+      when: telemetry_certs['server_csr'] is defined
+
+    - name: read dsmsroot key
+      set_fact:
+          dsmsroot_key: "{{ telemetry_certs['dsmsroot_key'] }}"
+      when: telemetry_certs['dsmsroot_key'] is defined
+
+    - name: read dsmsroot csr
+      set_fact:
+          dsmsroot_csr: "{{ telemetry_certs['dsmsroot_csr'] }}"
+      when: telemetry_certs['dsmsroot_csr'] is defined
+
+    - name: Create telemetry server private key
+      openssl_privatekey:
+          path: "{{ server_key }}"
+          size: 2048
+      become: true 
+
+    - name: create telemetry server csr
+      openssl_csr:
+          path: "{{ server_csr }}"
+          privatekey_path: "{{ server_key }}"
+      become: true
+    
+    - name: Generate a Self Signed OpenSSL telemetry server certificate
+      openssl_certificate:
+          path: /etc/sonic/telemetry/streamingtelemetryserver.cer
```

Comment:
why not use variable for this one as other ones?

Reply:
This one has not been reused. Do we need a variable in that case?

Reply:
added variable